easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.4.5-native.eb

```diff
@@ -0,0 +1,30 @@
+name = 'Hadoop'
+version = '2.6.0-cdh5.4.5'
+versionsuffix = '-native'
+
+homepage = 'http://archive.cloudera.com/cdh5/cdh/5/'
+description = """Hadoop MapReduce by Cloudera"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-src.tar.gz']
+source_urls = ['http://archive.cloudera.com/cdh5/cdh/5/']
+
+patches = ['Hadoop-TeraSort-on-local-filesystem.patch']
+
+builddependencies = [
+    ('Maven', '3.3.3'),
+    ('protobuf', '2.5.0'),  # *must* be this version
+    ('CMake', '3.3.1'),
+    ('snappy', '1.1.3', '', ('GCC', '4.9.3')),
+]
+
+dependencies = [('Java', '1.7.0_80')]
```

Comment:
This is the latest Java 7 release, and it seems like Java 8 + Hadoop is not a good idea yet, see https://issues.apache.org/jira/browse/HADOOP-11090